posix.mak

```diff
@@ -504,40 +505,40 @@ else
 	DMD_EXCLUDE += -e /scanmach/d -e /libmach/d
 endif
 
-docs.json : ${DMD} ${DMD_REL} ${DMD_STABLE_DIR} ${DRUNTIME_STABLE_DIR} \
+$G/docs.json : ${DMD} ${DMD_REL} ${DMD_STABLE_DIR} ${DRUNTIME_STABLE_DIR} \
 		${PHOBOS_STABLE_FILES_GENERATED} | dpl-docs
 	find ${DMD_STABLE_DIR}/src -name '*.d' | \
 		sed -e /mscoff/d -e /objc_glue.d/d -e /objc.d/d ${DMD_EXCLUDE}  \
-			> .release-files.txt
+			> $G/.release-files.txt
 	find ${DRUNTIME_STABLE_DIR}/src -name '*.d' | \
-	  sed -e /unittest.d/d -e /gcstub/d >> .release-files.txt
+	  sed -e /unittest.d/d -e /gcstub/d >> $G/.release-files.txt
 	find ${PHOBOS_STABLE_DIR_GENERATED} -name '*.d' | \
-	  sed -e /unittest.d/d -e /windows/d | sort >> .release-files.txt
+	  sed -e /unittest.d/d -e /windows/d | sort >> $G/.release-files.txt
 	${DMD_REL} -J$(DMD_STABLE_DIR)/res -J$(dir $(DMD_REL)) -c -o- -version=CoreDdoc \
-		-version=MARS -version=CoreDdoc -version=StdDdoc -Df.release-dummy.html \
-	  -Xfdocs.json -I${PHOBOS_STABLE_DIR_GENERATED} @.release-files.txt
-	${DPL_DOCS} filter docs.json --min-protection=Protected \
+		-version=MARS -version=CoreDdoc -version=StdDdoc -Df$G/.release-dummy.html \
+	  -Xf$@ -I${PHOBOS_STABLE_DIR_GENERATED} @$G/.release-files.txt
+	${DPL_DOCS} filter $@ --min-protection=Protected \
 	  --only-documented $(MOD_EXCLUDES_PRERELEASE)
-	rm .release-files.txt .release-dummy.html
+	rm $G/.release-files.txt $G/.release-dummy.html
```

Comment:
In theory now that it's in $(GENERATED), we could drop this line (though it doesn't hurt either).